Apsitvarkom.Api/Controllers/GeocodingController.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Apsitvarkom.DataAccess;
+using Apsitvarkom.Models;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Apsitvarkom.Api.Controllers
+{
+    [ApiController]
+    [Route("/api/[controller]")]
+    public class GeocodingController
+    {
+        private readonly IGeocoder _geocoder;
+
+        public GeocodingController(IGeocoder geocoder)
+        {
+            _geocoder = geocoder;
+        }
+        [HttpGet]
+        [ProducesResponseType(typeof(string), StatusCodes.Status200OK)]
+        public async Task<string> getGeocoder()
```

Comment:
PascalCase for methods, i. e. `GetGeocoder`. Even better would be `ReverseGeocode` since it describes what the action is used for